homeassistant/components/switcher_kis/quality_scale.yaml

```diff
@@ -0,0 +1,78 @@
+rules:
```

Comment:
```py
    @callback
    def _handle_coordinator_update(self) -> None:
        """Handle updated data from the coordinator."""
        self._update_data()
        self.async_write_ha_state()
```
consider moving this to the base entity and just have a stub `self._update_data` there

Reply:
https://github.com/home-assistant/core/pull/143738